index.html

```diff
@@ -7,21 +7,27 @@
     <link rel="stylesheet" href="styles.css">
     <title>Kiplaacollo Portfolio website</title>
   </head>
-  <body>
+  <body>        
+    <header>
+      <nav class="navbar">
+        <a href="#" class="logo head">Welcome</a>
```

Comment:
Kindly fix the overflowing issues as the previous reviewer requested. Now the hamburger button is located better but your app still has some width problems. You can start fixing them by removing the 8px margin around your body 👍
Right now still looks like below
![Screen Recording 2023-01-03 at 9 55 42 PM](https://user-images.githubusercontent.com/25887253/210424056-c24525fb-0470-4118-b744-c8204984d419.gif)